server/sql.go

```diff
@@ -411,6 +411,11 @@ func (s *Server) syncEngineVersionAndSchema(ctx context.Context, instance *api.I
 			}
 
 			for _, schema := range schemaList {
+				schemaVersion, err := getLatestSchemaVersion(ctx, driver, schema.Name)
```

Comment:
Let's leave a comment here suggesting this might have performance issue (N+1) and will cause frontend timeout since we set a 30s limit (INSTANCE_OPERATION_TIMEOUT)

This would happen when there is long RT and we have some databases.